### PR TITLE
DR-889 update skaffold template chart version

### PR DESCRIPTION
**Pull request by marikomedlock**
- Updated chart versions in skaffold template yaml.
- Added a section to the README about how to add/override a new application property.
- Added a comment to db-connect about how to set the environment variables before calling.

I'd especially appreciate @smark88 review of the README. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +156,46 @@ Run a deployment you must set env var `IMAGE_TAG`
...
+            - name: DATAREPO_GCS_ALLOWREUSEEXISTINGBUCKETS
+              value: {{ .Values.env.datarepoGcsAllowreuseexistingbuckets | quote }}
+            {{- end }}
+    ```
+    - Find the the [values.yaml](https://github.com/broadinstitute/datarepo-helm/blob/master/charts/datarepo-api/values.yaml) file.
+    - Add a new line under the `env` section.
+    ```
+      datarepoGcsAllowreuseexistingbuckets:
+    ```
+    - Release a new version of the chart. Talk to DevOps/Mark S to do this.
```
Comment (fboulnois) [inline README.md]:
```suggestion
    - Release a new version of the chart. Talk to DevOps to do this.
```

Comment (fboulnois) [reply README.md]: Based on conversations I had with Jeremy for the getting started guide, we've avoided calling out specific people.

**Review by fboulnois — CHANGES_REQUESTED** [review]
I appreciate keeping the documentation up to date!

**Review by thathert — APPROVED** [review]
Thanks for updating! LGTM assuming we remove Mark's name there.

File: README.md (hunk 1 above)
Comment (marikomedlock) [reply README.md]: Done, thanks.

**Review by fboulnois — APPROVED** [review]
lgtm 👍

**Pull request merged**